data/catalogs/deltares_data/v0.7.0/data_catalog.yml

```diff
@@ -0,0 +1,1881 @@
+
+ghs_pop_2015_54009_v2019a:
+  crs: "ESRI:54009"
+  data_type: RasterDataset
+  driver: raster
+  kwargs:
+    chunks:
+      x: 1000
+      y: 1000
+  meta:
+    category: socio-economic
+    paper_doi: 10.2760/062975
+    paper_ref: Florczyk et al (2019)
+    source_license: CC BY 4.0
+    source_url: https://ghsl.jrc.ec.europa.eu/download.php?ds=pop
+  path: socio_economic/ghs/GHS_POP_E2015_GLOBE_R2019A_54009_250_V1_0.tif
+
+ghs_smod:
+  crs: "ESRI:54009"
+  data_type: RasterDataset
+  driver: raster
+  kwargs:
+    chunks:
+      x: 3600
+      y: 3600
+  meta:
+    category: socio-economic
+    paper_ref: Pesaresi and Freire (2016)
+    source_author: JRC-ISPRA EC
+    source_license: https://data.jrc.ec.europa.eu/licence/com_reuse
+    source_url: https://data.jrc.ec.europa.eu/dataset/jrc-ghsl-ghs_smod_pop_globe_r2016a
+    source_version: R2016A_v1.0
+  path: socio_economic/ghs/GHS_SMOD_POP2015_GLOBE_R2016A_54009_1k_v1_0.tif
+
+ghs_smod_2015_54009_v2019a:
+  crs: "ESRI:54009"
+  data_type: RasterDataset
+  driver: raster
+  kwargs:
+    chunks:
+      x: 3600
+      y: 3600
+  meta:
+    category: socio-economic
+    paper_doi: 10.2905/42E8BE89-54FF-464E-BE7B-BF9E64DA5218
+    paper_ref: Pesaresi et al (2019)
+    source_author: JRC-ISPRA EC
+    source_license: https://data.jrc.ec.europa.eu/licence/com_reuse
+    source_url: https://data.jrc.ec.europa.eu/dataset/42e8be89-54ff-464e-be7b-bf9e64da5218
+    source_version: R2019A_v2.0
+  path: socio_economic/ghs/GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K_V2_0.tif
+
+globcover:
+  crs: 4326
+  data_type: RasterDataset
+  driver: raster
+  kwargs:
+    chunks:
+      x: 3600
+      y: 3600
+  meta:
+    category: landuse
+    paper_doi: 10.1594/PANGAEA.787668
+    paper_ref: Arino et al (2012)
+    source_license: CC-BY-3.0
+    source_url: http://due.esrin.esa.int/page_globcover.php
+    source_version: v2.3
+  path: landuse/globcover/GLOBCOVER_L4_200901_200912_V2.3.tif
+
+glofas_era5_v31:
+  crs: 4326
+  data_type: RasterDataset
+  meta:
+    category: hydro
+    notes: Extracted from Copernicus Climate Data Store
+    paper_doi: 10.5194/essd-12-2043-2020
+    paper_ref: Harrigan et al. (2020)
+    source_license: https://cds.climate.copernicus.eu/api/v2/terms/static/cems-floods.pdf
+    source_url: https://doi.org/10.24381/cds.a4fdd6b9
+    source_version: v31
+  variants:
+    - provider: normal
+      driver: netcdf
+      kwargs:
+        chunks:
+          time: 180
+          latitude: 150
+          longitude: 360
+        combine: by_coords
+        decode_times: true
+        parallel: true
+      path: hydro/glofas_era5/glofas_v31_{year}.nc
+      rename:
+        dis24: discharge
+    - provider: uparea
+      driver: raster
+      kwargs:
+        chunks:
+          x: 150
+          y: 360
+      path: hydro/glofas_era5/glofas_v31_uparea.tif
+      rename:
+        glofas_v31_uparea: uparea
+      unit_mult:
+        uparea: 0.000001
```

Comment:
@DirkEilander I migrated an alias here to a new variant, or do we want these to be different entries?